defaults/main.yml

```
@@ -5,6 +5,8 @@ bbb_upgrade_packages: "{{ 'yes' if bbb_state == 'latest' else 'no' }}"
 bbb_check_for_running_meetings: true
 bbb_letsencrypt_enable: true
 bbb_letsencrypt_api: https://acme-v02.api.letsencrypt.org/directory
+bbb_nginx_install_from_source: true
+bbb_nginx_branch: 'mainline'
```

I would rather like to default this to 'stable', since the current stable version (1.20) is already much newer than bionic default (1.14) and also supports TLS 1.3.

```suggestion
bbb_nginx_branch: 'stable'
```

---

One may go with the stable release but the mainline release is very stable as well. You may write any other word into 'bbb_nginx_branch' and the "stable" package will be deployed. 

`deb https://nginx.org/packages/{{ (bbb_nginx_branch == 'mainline') | ternary('mainline/', '') }}`

---

I added a switch to enable mainline (if required)

---

It seems there is no 'stable' version available via the official nginx repo. Adding a community ppa might be our only option. But then again this seems a bit overhead. What do you think? Which ppa to use then?

---

Yes there is a official stable repo:
https://nginx.org/packages/ubuntu/

---

See also https://nginx.org/en/linux_packages.html#Ubuntu